lib/_tls_common.js

```diff
@@ -162,6 +162,13 @@ exports.createSecureContext = function createSecureContext(options, context) {
     c.context.setFreeListLength(0);
   }
 
+  if (options.clientCertEngine) {
+    if (c.context.setClientCertEngine)
+      c.context.setClientCertEngine(options.clientCertEngine);
```

Comment:
This should typcheck that `options.clientCertEngine` is a string

Reply:
@addaleax Fixed (hopefully in a way that is acceptable). Thanks.